Automattic/wp-calypso
# Logged Out Theme Showcase: Add new categories to SEO map

Related to https://github.com/Automattic/dotcom-forge/issues/3311

## Proposed Changes

* Added the new Categories to the SEO map that we've added to the Logged Out Theme Showcase.

| Before | After |
|--------|--------|
|![Captura de Pantalla 2023-08-15 a las 14 09 53](https://github.com/Automattic/wp-calypso/assets/1989914/b0ccc17c-1a0b-402f-99d4-4c44b892bc98)|![Captura de Pantalla 2023-08-15 a las 14 10 04](https://github.com/Automattic/wp-calypso/assets/1989914/03c80612-0ca4-4744-93cf-042d154c074e)| 

## Testing Instructions



* While logged-out, visit `/themes`.
  * If you are running Calypso locally, ensure [the SSR is in debug mode](https://github.com/Automattic/wp-calypso/blob/trunk/docs/server-side-rendering.md#debugging).
* Change subjects and pricing filters.
* Check the page's heading and sub-heading, and ensure the copy matches the hardcoded list provided in https://docs.google.com/spreadsheets/d/1dulcaxW-zORE7RvDMZEUHt86AFvf9hzgtHXYpUUHMRY/edit?usp=sharing.
* Inspect the page source, and ensure the `<title>` and `<meta name="description">` tags too match the hardcoded list.
* Open `/themes` while logged-in.
* Ensure the header looks like this and does not changes depending on the filters.
  <img width="416" alt="Screenshot 2023-07-25 at 18 48 45" src="https://github.com/Automattic/wp-calypso/assets/2070010/44ec70c1-8f14-4064-a4d4-3da116e28bac">


## Pre-merge Checklist
